mismi-s3/test/test-io.hs

```diff
@@ -2,11 +2,13 @@ import           Disorder.Core.Main
 
 import qualified Test.IO.Mismi.S3.Commands
 import qualified Test.IO.Mismi.S3.Internal
-
+import           Test.Mismi (enableTests)
 
 main :: IO ()
 main =
-  disorderMain [
-      Test.IO.Mismi.S3.Commands.tests
-    , Test.IO.Mismi.S3.Internal.tests
+  disorderMain =<< enableTests "AWS_TEST_S3" [
+      Test.IO.Mismi.S3.Internal.tests
+    ] [
+      Test.IO.Mismi.S3.Internal.tests
```

Comment:
Would it make sense to always include the tests from the first list in the second? /notsure